src/app/state/locales/en/landing.json

```
@@ -21,5 +21,10 @@
   "delegate_your_xrd_to_our": "Delegate your $XRD to our Validator to earn $DEXTR.",
   "earn_dextr_by_contributing": "Earn $DEXTR by contributing",
   "whether_you_are_a_developer": "Whether you're a developer, designer or marketing enthusiast, all contributors get rewarded in $DEXTR tokens. We are 100% community build with no formal team.",
-  "join_us": "Join us"
+  "join_us": "Join us",
+  "pairs": "Pairs",
+  "users_trust_us": "users trust us",
```

I know it doesnt make any difference as the frontend will UPPERCASE this text, but to be consisten please capitalize `Users` to follow our formatting guideline.

```suggestion
  "users_trust_us": "Users trust us",
```

---

my mistake, it makes sense ! :)